includes/components/html/new/ok-msg.php

```diff
@@ -78,4 +78,22 @@ function handleChange() {
         update(toggle.value);
     });
 
+    const initialValue = localStorage.getItem("recentClips");
+    const url = "<?php echo $url ?>";
+
+    if (initialValue) {
+        const recentlyMadeArray = JSON.parse(initialValue);
+        if (!recentlyMadeArray.includes(url)) {
+            recentlyMadeArray.push(url);
+        }
+        if (recentlyMadeArray.length > 6) {
+            const reversedRecents = recentlyMadeArray.reverse();
+            reversedRecents.pop();
+            localStorage.setItem("recentClips", JSON.stringify(reversedRecents.reverse()));
+        } else {
+            localStorage.setItem("recentClips", JSON.stringify(recentlyMadeArray));
+        }
+    } else {
+        localStorage.setItem("recentClips", JSON.stringify([url]))
+    }
```

Comment:
No return statements 🤔

Reply:
I don't think there shouldn't be any, or should there? It is not in any function, so it cannot return